src/DeviceFirmata.cpp

```diff
@@ -0,0 +1,264 @@
+/*
+  DeviceFirmata.cpp - Firmata library
+*/
+
+#include "DeviceFirmata.h"
+#include "utility/Boards.h"
+#include <Base64.h>
```

Comment:
My assumption is availability of Base64 libraries for client applications, but wanted to understand if there are other reasons/advantages.